addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -504,13 +514,22 @@ odoo.define('website_slides.fullscreen', function (require) {
             });
             this.set('slide', newSlide);
         },
+        /**
+         * Trigger when a slide has been completed by other component and than need
+         * the UI to be updated.
+         *
+         * @private
+         */
+        _onSlideCompleted: function (ev) {
+            this._markAscompleted(ev.data.slide.id, ev.data.completion);
+        },
         /**
          * Triggered when sub widget business is done and that slide
          * can now be marked as done.
          *
          * @private
          */
-        _onSlideCompleted: function (ev) {
+        _onSlideToComplete: function (ev) {
```

Comment:
Method name is a little weird.
`_onSlideSetCompleted` ?

Reply:
I want to differenciate 
- slide mark as done on the server and that need to be updated on the UI (slide_completed)
- slide to be mark as completed from the js to the server (slide_to_complete)
I don't know if it's clear enough .... :s

Reply:
That's why I suggested slide_set_complete, which sounds more like an action.
(slide_to_complete sounds more like a variable).

Sounds better for me but at this point it could be a preference thing, I let you choose.